docs/UserGuide.md

```diff
@@ -92,15 +92,19 @@ Examples:
 
 Adds a new record to a given patient.
 
-Format: `addR INDEX d/DATE r/DESCRIPTION`
+Format: `addR d/DATE r/DATA [m/MEDICATION]…` 
 
-* Adds a new record to the patient at the specified `INDEX`.
-* The index refers to the index number shown in the displayed patient records list.
-* The index must be a positive integer 1, 2, 3, …​
-* The record is created with given `DATE` and `DESCRIPTION` information.
+* The command is only valid after using `listR PATIENT_INDEX`.
+* Adds a new record to the patient with given `DATE`, `DATA` and `MEDICATION` information.
```

Comment:
It would be useful to indicate that the Medication field is optional, unlike the date and record string.

Reply:
Good point, will update in future commit